chapters/5-processing.tex

```diff
@@ -42,265 +46,354 @@
+the data and remove duplicated rows
+until each data table is uniquely and fully identified by the identifying variable
 that corresponds to its unit of observation.
+Reshaping data tables is the most intricate task in data cleaning;
+you should be very familiar with commands such as
+\texttt{reshape} in Stata and \texttt{pivot} in R.
+You must be sure that identifying variables are consistent across data tables,
+so they can always be linked.
+Reshaping is the type of transformation we referred to
+in the example of how you calculate 
+the share of women in a wide data set. 
+The important difference is that 
+in a tidy workflow,
+instead of transforming the data for each operation,
+this transformation is done once for all data during cleaning,
+making all subsequent operations much easier.
+
 In the earlier household survey example,
-household-level variables will be stored in one tidy dataset, 
-and household-member variables are reshaped to member level and stored in another tidy dataset,
+household-level variables will be stored in one tidy data table,
+and household-member variables are reshaped to member level
+and stored in another tidy data table,
 which also contains the household ID for each individual.
-So the household ID is intentionally duplicated in the household members table.
+The household ID is intentionally duplicated in the household members data table
+to allow one or several household members to be linked to the same household data.
+The household members data will be uniquely identified by
+either a single household member ID or
+a combination of household ID and household member ID.
 In the transaction data example,
-the result of the tidying process would be one transaction-level dataset, 
+the result of the tidying process would be one transaction-level data table,
 containing variables indicating the ID of all firms involved;
-and one firm-level dataset with a single entry for each firm.
-
-Reshaping tables is the most intricate task in this process,
-so you should be very familiar with commands such as \texttt{reshape} in Stata and \texttt{pivot} in R.
-Another key aspect in this process is to make sure that ID variables are consistent across tables,
-so they can always be linked.
-There must be a clear way to connect each tidy table to a master dataset.
-Note that the tidying process gets more complex as the number of nested groups increases.
+and one firm-level data table with a single entry for each firm.
+Then, firm-level analysis is easily done
+by calculating appropriate statistics in the transactions data table
+(in Stata, often through \texttt{collapse})
+and then merging or joining those results to the firms data table.
+
+In both examples above, your dataset is made up of two data tables,
+each saved in a separate file.
+In a tidy workflow, your dataset is a set of one or more data tables,
+where each data table is a file in your data storage.
```

Comment:
```suggestion
In a tidy workflow, your clean dataset is a set of one or more tidy data tables. In both examples above, your clean dataset is made up of two tidy data tables.

```

Reply:
I didn't understand what "where each data table is a file in your data storage" means. i removed but if you want to add back in should clarify